intro-to-object-oriented-programming/instance-methods.md

```diff
@@ -86,6 +86,18 @@ In the above example, the instance method `get_audio_data` takes an additional p
 
 Inside the `get_audio_data` method, we use the attribute `self.track_list` _**and**_ the argument `track_index`. We assume `self.track_list` is a list, and we use it in a conditional statement. We use `track_index` in the line `return self.track_list[track_index - 1]`.
 
+### callout-info
+
+## Instances of a Class do not share State but they do share Behavior
+
+As we've been learning, the two building-blocks of a class are **State** (ie. its attributes) and **Behavior** (ie. its methods). 
+
+Like we saw in the [Creating Instances lesson](creating-instances.md), **each instance of a Class has its own state**. This means that one album object can have the title "When Doves Cry" and another album object have the title "Dirty Computer". 
+
+We will see now that **all instances of a class have the same behavior**. This means that the method `get_audio_data` will be the same for all album objects. The only thing that will change from one instance to another is the value of the `track_list` attribute.
+
+### end-callout
```

Comment:
```suggestion
### !callout-info

## Instances of a Class do not share State but they do share Behavior

As we've been learning, the two building-blocks of a class are **state** (ie. its attributes) and **behavior** (ie. its methods). 

Like we saw in the [Creating Instances lesson](creating-instances.md), **each instance of a Class has its own state**. This means that one album object can have the title "When Doves Cry" and another album object have the title "Dirty Computer". 

We will see now that **all instances of a class have the same behavior**. This means that the method `get_audio_data` will be the same for all album objects. The only thing that will change from one instance to another is the value of the `track_list` attribute.

### !end-callout
```

The call-out was missing the '!' and did a lower case for 'state' and 'behavior' for consistency with other lessons. Overall, this was an important call out and I appreciate you writing this portion Jared!

Reply:
Oh no! Thank you for catching that! I clearly didn't do a final preview with my last changes of everything. I should do that now